src/redux/sagas/RatingSaga/RatingSaga.js

```diff
@@ -0,0 +1,34 @@
+import { put, call, takeLatest } from 'redux-saga/effects';
+import axios from 'axios';
+import * as types from '../../actions/RatingsActions/types';
+
+function* ratingWorker({ payload }) {
```

Comment:
Can this have a name better than ratingWorker?

Reply:
we can have another name here. am calling it rating worker owing to its function.